src/components/FileRepo/AggregationSidebar/AggregationSidebar.js

```diff
@@ -107,35 +129,50 @@ const AggregationSidebar = compose(injectState, withTheme, withApi)(
                   })
                 }
               >
-                <Trans>All Filters</Trans>
-              </TealActionButton>
-            </AggregationHeader>
-          </Controls>
-          <CustomAggregationsPanel
-            {...{
-              ...props,
-              state,
-              effects,
-              setSQON,
-              containerRef: aggregationsWrapperRef,
-              translateSQONValue,
-              onValueChange: ({ active, field, value }) => {
-                if (active) {
-                  trackFileRepoInteraction({
-                    category: TRACKING_EVENTS.categories.fileRepo.filters,
-                    action: 'Filter Selected',
-                    label: { type: 'filter', value, field },
-                  });
-                }
-              },
-              componentProps: {
-                getTermAggProps: () => ({
-                  InputComponent: FilterInput,
-                }),
-              },
-            }}
-          />
-        </AggregationWrapper>
+                <Trans>Browse All</Trans>
+              </BrowseAllButton>
+            ) : null}
+            {!expanded ? (
+              <OpenActionChevron
+                width={14}
+                onClick={() => setExpanded(!expanded)}
+                fill={theme.secondary}
+              />
+            ) : (
+              <CloseActionChevron
+                onClick={() => setExpanded(!expanded)}
+                width={14}
+                fill={theme.secondary}
+              />
+            )}
+          </AggregationHeader>
+          {expanded ? (
```

Comment:
so instead of the conditional rendering, we can do:
```
<CollapsibleDiv expanded={expanded}/>
   <CustomAggregationsPanel {...stuff}/>
</CollapsibleDiv>
```
where:
```
const CollapsibleDiv = styled('div')`
   display: ${({ expanded }) => (expanded ? `block` : `none`)};
   height: 100%;
`
```
The div immediately after the Tabs probably needs a `height:100%` too, I think

The cleaner way of going through all this is probably feeding the `expanded` state into `CustomAggregationsPanel` as a `hidden` prop and have it hide itself internally